projects/1.19/assets/storage-racks/storageracks/lang/zh_cn.json

```
@@ -0,0 +1,133 @@
+    "block.storageracks.mangrove_gold_rack": "红树木金架子",
+    "block.storageracks.mangrove_iron_corner": "红树木铁转角",
+    "block.storageracks.mangrove_iron_rack": "红树木铁架子",
+    "block.storageracks.mangrove_stone_corner": "红树木石转角",
+    "block.storageracks.mangrove_stone_rack": "红树木石架子",
+    "block.storageracks.mangrove_wood_corner": "红树木转角",
+    "block.storageracks.mangrove_wood_rack": "红树木架子",
+    "block.storageracks.oak_diamond_corner": "橡木钻石转角",
+    "block.storageracks.oak_diamond_rack": "橡木钻石架子",
+    "block.storageracks.oak_emerald_corner": "橡木绿宝石转角",
+    "block.storageracks.oak_emerald_rack": "橡木绿宝石架子",
+    "block.storageracks.oak_gold_corner": "橡木金转角",
+    "block.storageracks.oak_gold_rack": "橡木金架子",
+    "block.storageracks.oak_iron_corner": "橡木铁转角",
+    "block.storageracks.oak_iron_rack": "橡木铁架子",
+    "block.storageracks.oak_stone_corner": "橡木石转角",
+    "block.storageracks.oak_stone_rack": "橡木石架子",
+    "block.storageracks.oak_wood_corner": "橡木转角",
+    "block.storageracks.oak_wood_rack": "橡木架子",
+    "block.storageracks.spruce_diamond_corner": "云杉木钻石转角",
+    "block.storageracks.spruce_diamond_rack": "云杉木钻石架子",
+    "block.storageracks.spruce_emerald_corner": "云杉木绿宝石转角",
+    "block.storageracks.spruce_emerald_rack": "云杉木绿宝石架子",
+    "block.storageracks.spruce_gold_corner": "云杉木金转角",
+    "block.storageracks.spruce_gold_rack": "云杉木金架子",
+    "block.storageracks.spruce_iron_corner": "云杉木铁转角",
+    "block.storageracks.spruce_iron_rack": "云杉木铁架子",
+    "block.storageracks.spruce_stone_corner": "云杉木石转角",
+    "block.storageracks.spruce_stone_rack": "云杉木石架子",
+    "block.storageracks.spruce_wood_corner": "云杉木转角",
+    "block.storageracks.spruce_wood_rack": "云杉木架子",
+    "block.storageracks.stone_controller": "石控制器",
+    "block.storageracks.warped_diamond_corner": "诡异木钻石转角",
+    "block.storageracks.warped_diamond_rack": "诡异木钻石架子",
+    "block.storageracks.warped_emerald_corner": "诡异木绿宝石转角",
+    "block.storageracks.warped_emerald_rack": "诡异木绿宝石架子",
+    "block.storageracks.warped_gold_corner": "诡异木金转角",
+    "block.storageracks.warped_gold_rack": "诡异木金架子",
+    "block.storageracks.warped_iron_corner": "诡异木铁转角",
+    "block.storageracks.warped_iron_rack": "诡异木铁架子",
+    "block.storageracks.warped_stone_corner": "诡异木石转角",
+    "block.storageracks.warped_stone_rack": "诡异木石架子",
+    "block.storageracks.warped_wood_corner": "诡异木转角",
+    "block.storageracks.warped_wood_rack": "诡异木架子",
+    "com.storageracks.insert.unlock.succeeded": "成功解锁了输入功能",
+    "com.storageracks.insert.unlock.failed": "解锁输入失败，你的物品栏里缺少1个漏斗。",
+    "com.storageracks.sort.unlock.failed": "解锁排序失败，你的物品栏里缺少1个红石块。",
+    "com.storageracks.sort.unlock.succeeded": "成功解锁了排序功能",
+    "container.title.insertion": "输入控制器",
+    "container.title.rack": "架子",
+    "gui.storage.racks.available": "物品栏中可用",
+    "gui.storage.racks.missing": "缺少所需物品",
+    "gui.storageracks.allinventory": "存储",
+    "gui.storageracks.doublecontroller": "每个网络只能存在一个控制器！",

```

| **审查级别**    | 备注 |
|--------------|----|
| ⚪ 轻微 / 不阻止合并但需三思  |    |

原文：**There can only be 1 Controller per Network!**
```suggestion
    "gui.storageracks.doublecontroller": "每个网络只能存在1个控制器！",
```
与L117-118统一，或者都译作大写中文数字